dvf_json/src/Plugin/Visualisation/Source/JsonFile.php

```diff
@@ -231,6 +232,10 @@ protected function fetchData() {
     $data = '{}';
 
     if ($response) {
+      $dvfHelpers = new DvfHelpers();
+      if(!$dvfHelpers->validateJson($response)) {
+        $this->logger->error($this->t('This is invalid JSON file'));
```

Comment:
I suggest rewording to `This JSON file is invalid`

Reply:
Done